projects/soft_999_eddy_kim_fw_103/src/main.c

```diff
@@ -0,0 +1,49 @@
+#include "adc.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+static void callback(const GpioAddress *address, void *context) {
+  // Is it right to pass the GpioAddress A6 by *context? I'm unsure what *address is supposed to
+  // return
+  GpioAddress *A6 = context;
+  uint16_t A6_data = 0;
+  adc_read_raw_pin(*A6, &A6_data);
```

Comment:
The homework does say to use a _converted_ read, so use `adc_read_converted_pin`. The difference is that a converted read converts the value to millivolts, whereas a raw read is just whatever the ADC spits out.